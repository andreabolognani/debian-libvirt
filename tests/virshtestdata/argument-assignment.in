setvcpus test 2
setvcpus --domain test 2
setvcpus --domain=test 2
setvcpus test --count 2
setvcpus test --count=2
setvcpus --domain test --count 2
setvcpus --domain=test --count 2
setvcpus --domain test --count=2
setvcpus --domain=test --count=2
setvcpus --count 2 --domain test
setvcpus --count 2 --domain=test
setvcpus --count=2 --domain test
setvcpus --count=2 --domain=test
setvcpus --count 2 test
setvcpus --count=2 test
