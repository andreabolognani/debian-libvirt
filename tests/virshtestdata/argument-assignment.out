commands: "setvcpus test 2
setvcpus --domain test 2
setvcpus --domain=test 2
setvcpus test --count 2
setvcpus test --count=2
setvcpus --domain test --count 2
setvcpus --domain=test --count 2
setvcpus --domain test --count=2
setvcpus --domain=test --count=2
setvcpus --count 2 --domain test
setvcpus --count 2 --domain=test
setvcpus --count=2 --domain test
setvcpus --count=2 --domain=test
setvcpus --count 2 test
setvcpus --count=2 test
"
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: count(optdata): 2
setvcpus: domain(optdata): test
setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

setvcpus: found option <domain>: test
setvcpus: <domain> trying as domain NAME

